lerobot/common/policies/vla/modeling_vla.py

```diff
@@ -91,6 +92,7 @@ def forward(self, batch: dict[str, torch.Tensor]) -> dict[str, torch.Tensor]:
         if len(self.expected_image_keys) > 0:
             batch = dict(batch)
             batch["observation.images"] = [img for k in self.expected_image_keys for img in batch[k]]
+        batch = self.normalize_targets(batch)
```

Comment:
I am still testing the code to verify that the reward 0 was indeed related to not normalizing targets.